certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -1368,91 +1410,101 @@ PROCEDURE VALIDATEACCOUNT
 **  RETURN  USEROPTIONSMAXUSED            Highest Option number avail for cert
 *]
  INVALIDACCTTYPEFOUND=FALSE
- IF INVALIDACCTYPES(ACCOUNT:TYPE)=TRUE THEN
-  INVALIDACCTTYPEFOUND=TRUE
-
- USEROPTIONSMAXUSED=0
+ CROSSACCOUNTFLAG=FALSE
 
- FOR ACCOUNT TARGETACCOUNT
+ IF TARGETACCOUNT<>ACCOUNT:NUMBER THEN
   DO
-   WARNINGFOUND=FALSE
-   FOR TMPLOOP=1 TO MAXWARNINGTYPES
+   CROSSACCOUNTFLAG=TRUE
+   ERRORCODE=505
+  END
+ ELSE
+  DO
+   IF INVALIDACCTYPES(ACCOUNT:TYPE)=TRUE THEN
+    INVALIDACCTTYPEFOUND=TRUE
+
+   USEROPTIONSMAXUSED=0
+
+   FOR ACCOUNT TARGETACCOUNT
     DO
-     IF WARNINGSACCT(TMPLOOP)=TRUE AND
-        ANYWARNING(ACCOUNT,TMPLOOP) THEN
+     WARNINGFOUND=FALSE
+     FOR TMPLOOP=1 TO MAXWARNINGTYPES
       DO
-       WARNINGFOUND=TMPLOOP
-       TMPLOOP=MAXWARNINGTYPES
+       IF WARNINGSACCT(TMPLOOP)=TRUE AND
+          ANYWARNING(ACCOUNT,TMPLOOP) THEN
+        DO
+         WARNINGFOUND=TMPLOOP
```

Comment:
Looks like this will work (logically) but is it more clear to just say WARNINGFOUND=TRUE?

Reply:
In this case, by assigning WARNINGFOUND=TMPLOOP, WARNINGFOUND then contains the actual warning code which triggered the 502 error and can then later be easier referenced in case someone would want to know which targeted warning code actually tripped the error....

```
 IF ERRORDETAILOFFSET=2 THEN
  IF WARNINGFOUND<>FALSE THEN
   ERRORDETAIL(2)=ERRORDETAIL(2)+FORMAT(": Acct or Share Warning 999",WARNINGFOUND)
```

Reply:
Ah.  I missed that.  Thanks for the clarification.